group_vars/fender_cluster/vars.yml

```diff
@@ -76,18 +76,13 @@ nameservers: [
 ]
 local_admin_groups:
   - 'admin'
-  - 'docker'
 local_admin_users:
-  - 'egon'
   - 'gerben'
-  - 'henkjan'
   - 'marieke'
   - 'marloes'
-  - 'morris'
```

Comment:
why are we removing Morris as admin just on these two clusters?

Reply:
Account was never used: removed on redeploy from scratch.